tests/test_ossfs.py

```diff
@@ -27,22 +27,23 @@
 bucket = None
 get_oss_path = None
 
-def _have_required_env():
-  return os.getenv("OSS_CREDENTIALS") and os.getenv("OSS_FS_TEST_BUCKET")
+oss_id = None
+oss_key = None
+oss_host = None
 
-_msg = ("OSS tests skipped. To enable them, set OSS_CREDENTIALS env variable "
-        "to the path of your oss credential file and OSS_FS_TEST_BUCKET env "
-        "variable to your oss test bucket name.")
+_msg = ("OSS tests skipped. To enable them, set oss_id, oss_key, oss_host and bucket variable "
+        "to its real value")
+def check_oss_variable():
```

Comment:
Should this be an internal function, e.g. check_oss_variable -> _check_oss_variable?